jax/experimental/jax2tf/tests/primitive_harness.py

```diff
@@ -1659,9 +1659,14 @@ def _make_triangular_eigh_operand(shape, dtype, lower: bool, rng: Rng):
             devices="tpu",
             dtypes=[np.complex64, np.complex128]),
           Limitation(
-            "unimplemented", devices="cpu", dtypes=[np.float16]),
+            "unimplemented", devices="cpu",
```

Comment:
This is a fine change, but it can be done with a single `Limitation` that applies to all devices. 

I suspect that you figured it out, but this does more than disable the `jax2tf` tests (as the title of the PR says). This says that `eigh` is not implemented in JAX, indirectly this will disable also the jax2tf tests for eigh and other tests built on primitive harnesses.